docs/protocol.md

```diff
@@ -68,6 +68,33 @@
   </tr>
 </table>
 
+<h3>HELP</h3>
+
+<table>
+  <tr>
+    <td><strong>Description</strong></td>
+    <td colspan="2">Get a list of all available commands.</td>
+  </tr>
+  <tr>
+    <td><strong>Syntax</strong></td>
+    <td colspan="2"><code>HELP</code></td>
+  </tr>
+  <tr>
+    <td><strong>Comments</strong></td>
+    <td colspan="2">
+    </td>
+  </tr>
+  <tr>
+    <td><strong>Returns</strong></td>
+    <td><em>info</em></td>
+    <td><em>info</em> contains the available commands</td>
+  </tr>
+  <tr>
+    <td><strong>Example</strong></td>
+    <td colspan="2"><code>$ HELP</code><br /><code>You have the following commands available:</code><br /><code>...................GENERAL...................</code><br /><code>LOGIN username password</code><br /><code>QUIT</code><br /><code>............ACCOUNT ADMINISTRATION............</code><br /><code>DEFAULT account_name</code><br /><code>NEWACCOUNT name [DEFAULT]</code><br /><code>REGISTER username password</code><br /><code>.................TRANSACTIONS.................</code><br /><code>SHOWMYACCOUNTS</code><br /><code>DEPOSIT account_name amount</code><br /><code>PAY person amount</code><br /><code>Please note customers must be logged in to use some commands.</code></td>
```

Comment:
The example does not make sense for this command. The output is too long and not an example as it doesn't change.
Note that it says that you need to be logged in in the output which is incorrect.
The command also should return "SUCCESS or FAIL" probably.

Kindest Regards,
Jules